go-controller/pkg/ovn/pods_test.go

```diff
@@ -410,23 +410,20 @@ var _ = Describe("OVN Pod Operations", func() {
 				)
 				t.populateLogicalSwitchCache(fakeOvn)
 				mockAddNBDBError(ovntest.LogicalSwitchPortType, t.portName,
-					ovntest.LogicalSwitchPortPortSecurity,
+					ovntest.LogicalSwitchPortExternalId,
```

Comment:
why is this change needed?

Reply:
This change is needed because the port security check is done later in the code (after a bunch of other nbctl commands). Therefore the fexec matches fail because the commands are different. This simply keeps the injected error in the same place as the original code.

Reply:
Ok, can you please change the error string that is passed in to reflect that since we are not injecting a port security error anymore, but an error when setting external ids?

Reply:
yeah, will do. There is also a race here in this test between when the update retries and removing the error from the mock client. Will address that too in a few minutes.